### DISCUSSION TITLE
mlmym

### DISCUSSION THREAD

**Discussion by IRQBreaker** [ubergeek77/Lemmy-Easy-Deploy#70] [Ideas]
Hi,

Would it be possible to include something like mlmym.org as an alternative front end in Lemmy-Easy-Deploy?

**Comment by ubergeek77** [comment]
Hello! Good idea, thanks for asking :)

Sorry to disappoint, but I don't really have the time or capacity to support additional frontends for this project.

However, Lemmy-Easy-Deploy has a pretty flexible customization system, which would allow you to add additional deployments without breaking Lemmy-Easy-Deploy updates. That system is documented here:

https://github.com/ubergeek77/Lemmy-Easy-Deploy/blob/main/ADVANCED_CONFIGURATION.md

I personally use this system to add Ntfy to my deployment, so that "emails" generated by my Lemmy instance become push notifications on my phone. It's pretty neat!

You would probably want to override the default docker-compose template to launch that frontend, and override the default Caddyfile template to have a proper subdomain for it.

I won't be adding anything like this to the project officially, but if someone else wrote instructions for this, I'd be willing to start a Wiki page or Discussion post documenting all these "mods" 😅
